openspec/changes/archive/2026-03-06-feat-cli-score-display/specs/cli/spec.md

```
@@ -0,0 +1,13 @@
+## ADDED Requirements
+
+### Requirement: CLI scout 命令显示分数和颜色高亮
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)